src/applications/disability-benefits/all-claims/pages/medicalCareHistory.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+function medicalTreatmentRequiredCheck(errors, state, formData) {
```

Comment:
Can we move this to validations.js?

Reply:
Nvm Must be something broken in the build scripts:
![image](https://user-images.githubusercontent.com/22040793/48275157-d510ea80-e412-11e8-8d3f-7a61b59903d4.png)